canvas_modules/common-canvas/src/common-canvas/common-canvas.scss

```diff
@@ -44,18 +44,43 @@ $panel-border-color: $border-subtle-01;
 	position: relative; // This allows the State Tag to have positio: absolute
 }
 
+.right-flyout-container {
+	display: flex;
+	border-left: $panel-border-width solid $panel-border-color;
+	min-height: fit-content;
+	min-width: fit-content;
+}
+
 .left-flyout-panel,
 .right-flyout-panel {
 	// This combination of height and min-height enable scrolling in the child panel,
 	// I believe this is because it sets a specific height on the CSS grid cell,
 	// so the contents of the panel can adjust themselves to the height accordingly.
 	height: 0;
 	min-height: 100%;
+}
 
+// Allow content in child to expand on drag using flex
+.right-flyout-panel {
+	min-width: fit-content;
+	display: flex;
+	flex: 1;
 	user-select: none; /* Prevent elements from being selectable */
 	background-color: $layer-01;
 }
 
+.right-flyout-drag {
+	cursor: ew-resize;
+	flex: 0 0 0.5px;
+	border: $panel-border-color;
+	background: $layer-01;
+	transition: all 0.2s ease-in;
+	&:hover {
+		flex: 0 0 $spacing-01;
```

Comment:
What is this for - is it necessary?  I took it out and the right flyout seems to size OK. If this is needed to get Common Properties to display correctly? If so it ought to be part of the Common Properties CSS, not here.

Reply:
I have updated above to include `spacing-01` in `right-flyout-drag`. 

But the problem with above spacing is that when a `common properties` `Accordion` is highlighted on click there is a small gap between right flyout left border and common properties.
<img width="503" alt="Screenshot 2024-11-12 at 10 15 53 PM" src="https://github.com/user-attachments/assets/44e93d16-888a-48d0-9b46-5a4dc4301d8d">

I tried to resolve this using common properties css but this is visible in canvas so initially I set the width to `0.5px` so that the gap isn't visible and on hover it will take up `1px` width to match with bottom panel, but I have reverted it now and kept `spacing-01` for the `right-flyout-drag`.

https://github.com/user-attachments/assets/304382b9-5e7b-4cc4-ac8e-4faaaac47ddf

Reply:
If you are concerned about the visual appearance of the UI after you've implemented a change you should be asking @Analuisa-Del-Rivero1  for her decision on how she wants it to appear. Unfortunately your vide doesn't show what it looks like with the accordion highlighting.

Reply:
BTW It's probably best to ask question to Ana in the issue to keep all the design information discussion together in one place.

Reply:
Perhaps you could do a video (or two?) that shows both approaches and what the accordion looks like in each case and explain to her about the difficulty in hovering over the drag area ... and see what she decides.